CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.12)
 
-project(lcvm VERSION 0.10)
+project(lcvm VERSION 0.10.1)
```

Comment:
Do not cut a new version: The maintainer is the person in charge of cutting new version (i.e. remove the changes on CMakeLists.txt)